microcosm_fastapi/logging_data_map.py

```diff
@@ -14,6 +14,10 @@ class LoggingInfo:
     operation_name: Optional[str] = None
     function_name: Optional[str] = None
 
+    def is_empty(self):
+        if self.operation_name is None and self.function_name is None:
+            return True
```

Comment:
```suggestion
    def is_empty(self) -> bool:
        return self.operation_name is None and self.function_name is None:
```

neater to return False rather than None in the not-empty case

```

Reply:
Yep definitely better - thanks Tony 👍